### PR TITLE
cloud: update use-chat2query-api.md release chat2query v2

### PR DESCRIPTION
**Pull request by nizheming**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by jiajunhuang — APPROVED** [review]

File: tidb-cloud/use-chat2query-api.md [hunk 1]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
-1. In the left pane of [**Data Service**](https://tidbcloud.com/console/data-service), click the name of **Chat2Query System** to view its details.
+1. In the left pane of [**Data Service**](https://tidbcloud.com/console/data-service), click the name of **Chat2Query** to view its details.
 2. In the **Authentication** area, click **Create API Key**.
 3. In the **Create API Key** dialog box, enter a description and select a role for your API key.
 
-    The role is used to control whether the API key can read or write data to the clusters linked to the Data App. You can select the `ReadOnly` or `ReadAndWrite` role:
-
-    - `ReadOnly`: only allows the API key to read data, such as `SELECT`, `SHOW`, `USE`, `DESC`, and `EXPLAIN` statements.
-    - `ReadAndWrite`: allows the API key to read and write data. You can use this API key to execute all SQL statements, such as DML and DDL statements.
+   - `Chat2Query Admin Role`: allows the API key to management data summary, generate sql from a question, execute any SQL statement
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:50]:
```suggestion
   - `Chat2Query Admin Role`: allows the API key to manage data summary, generate SQL statements according to instructions, and execute any SQL statements.
```

File: tidb-cloud/use-chat2query-api.md [hunk 2]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+   - `Chat2Query Data Context Management Role`: only allows the API key to generate, update and delete data summary
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:51]:
```suggestion
   - `Chat2Query Data Context Management Role`: only allows the API key to generate, update, and delete data summary.
```

File: tidb-cloud/use-chat2query-api.md [hunk 3]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+   - `Chat2Query SQL Generate & Execute (ReadOnly) Role`: can generate sql from a question, but only allows the API key to execute SELECT SQL statement
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:52]:
```suggestion
   - `Chat2Query SQL Generate & Execute (ReadOnly) Role`: allows the API key to generate SQL statements based on provided instructions and execute  `SELECT` SQL statements.
```

File: tidb-cloud/use-chat2query-api.md [hunk 4]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+   - `Chat2Query SQL Generate & Execute (ReadWrite) Role`: can generate sql from a question, but only allows the API key to execute any SQL statement
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:53]:
```suggestion
   - `Chat2Query SQL Generate & Execute (ReadWrite) Role`: allows the API key to generate SQL statements based on provided instructions and execute any SQL statements.
```

File: tidb-cloud/use-chat2query-api.md [hunk 5]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
 
 5. Click the **Data Service** link in the message to access the Chat2Query API.
 
-    You can find that the **Chat2Query System** [Data App](/tidb-cloud/tidb-cloud-glossary.md#data-app) and its **Chat2Data** [endpoint](/tidb-cloud/tidb-cloud-glossary.md#endpoint) are displayed in the left pane.
+    You can find that the **Chat2Query** [Data App](/tidb-cloud/tidb-cloud-glossary.md#data-app) and its chat2data [endpoints](/tidb-cloud/tidb-cloud-glossary.md#endpoint) are displayed in the left pane.
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:40]:
```suggestion
    You will be directed to the [Data App](/tidb-cloud/tidb-cloud-glossary.md#data-app) page. On the page, you can find the **Chat2Query** Data App and a list of endpoints under the App.
```

File: tidb-cloud/use-chat2query-api.md [hunk 6]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+  "code": 200,
+  "msg": "",
+  "result": {
+    "data_summary_id": 481235,
+    "job_id": "79c2b3d36c074943ab06a29e45dd5887"
+  }
+}
+```
+
+The api is an async api, we should poll the job results in get job api by each 5 seconds until the job been successed.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:118]:
```suggestion
The API is asynchronous. You can pull the job results in the GET job API every 5 seconds until the job succeeds.
```

File: tidb-cloud/use-chat2query-api.md [hunk 7]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+  "result": {
+    "ended_at": 1699518950, // when the job finished, it's UNIX timestamp
+    "job_id": "79c2b3d36c074943ab06a29e45dd5887",  // ID of current job
+    "result": DataSummaryObject, // see the definition bellow
+    "status": "done" // status of the current job
+  }
+}
+```
+
+`DataSummaryObject` represents AI explored infomation of the given database, here is the structure of `DataSummaryObject`:
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:138]:
```suggestion
`DataSummaryObject` represents AI explored infomation of the given database. The following is a structure example of `DataSummaryObject`:
```

File: tidb-cloud/use-chat2query-api.md [hunk 8]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+    "id": 481235,
+    "name": "data-context",
+    "org_id": 30061,
+    "type": "data", // type of data summary, it's `data` for data summary
+    "updated_at": 1699518950
+  }
+}
+```
+
+**Update data summary result using data summary id when you have an better result**
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:210]:
```suggestion
**Update data summary result using data summary ID when you have a better result.**
```

File: tidb-cloud/use-chat2query-api.md [hunk 9]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+  "result": {
+    "ended_at": 1699523242,
+    "job_id": "103d0bf8d8564ec7b2c768d3180d8ab1",
+    "result": null,
+    "status": "done"
+  }
+}
+```
+
+when the `status` is `done`, the job had finished and description of table and columns had been updated.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:260]:
```suggestion
When the `status` is `done`, the job is finished and the descriptions of the table and columns are updated.
```

File: tidb-cloud/use-chat2query-api.md [hunk 10]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+{
+  "code": 200,
+  "msg": "",
+  "result": {
+    "job_id": "3966d5bd95324a6283445e3a02ccd97c"
+  }
+}
+```
+
+The api is an async api, we should poll the job results in get job api by each 5 seconds until the job been successed.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:290]:
```suggestion
The API is asynchronous. You can pull the job results in the GET job API every 5 seconds until the job succeeds.
```

File: tidb-cloud/use-chat2query-api.md [hunk 11]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+      "time_elapsed": 3.854671001434326
+    },
+    "status": "done"
+  }
+}
+```
+
+## Step 4. Call the Chat2Data v1 endpoint
+
+You can call chat2data v1 endpoint directly get sql generated result. this api will give a faster response but lower performance.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:336]:
```suggestion
You can call the chat2data v1 endpoint directly to get the SQL generated result. This API will give a faster response but with lower performance.
```

File: tidb-cloud/use-chat2query-api.md [hunk 12]
```diff
@@ -78,26 +353,26 @@ TiDB Cloud generates code examples to help you call an endpoint. To get the exam
...
+   - Replace the `<your instruction>` placeholder with the instruction you want AI to generate and execute SQL statements.
+   - Replace the `<your table name, optional>` placeholder with the table name you want to query. If you do not specify a table name, AI will query all tables in the database.
 
 > **Note:**
 >
 > Each Chat2Query Data App has a rate limit of 100 requests per day. If you exceed the rate limit, the API returns a `429` error. For more quota, you can [submit a request](https://support.pingcap.com/hc/en-us/requests/new?ticket_form_id=7800003722519) to our support team.
+> API Key with the role `Chat2Query Data Context Management Role` cannot call the Chat2Data v1 endpoint.
 
-The following code example is used to find the most popular GitHub repository from `sample_data.github_events` table:
+The following code example is used to count how many users in `sp500insight.users` table:
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:365]:
```suggestion
The following code example is used to count how many users in the `sp500insight.users` table:
```

File: tidb-cloud/use-chat2query-api.md [hunk 13]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+    "status": "done",
+    "summary": "This data source contains information about companies, indexes, and historical stock price data. It is used for financial analysis, investment decision-making, and market research in the finance domain.",
+    "summary_keywords": [
+        "users"
+    ],
+    "table_relationship": {}
+}
+```
+
+**Get data summary result using data summary id in anytime**
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:184]:
```suggestion
**Get the data summary result using the data summary ID.**
```

File: tidb-cloud/use-chat2query-api.md [hunk 14]
```diff
@@ -37,28 +37,303 @@ To enable the Chat2Query API, perform the following steps:
...
+    "job_id": "103d0bf8d8564ec7b2c768d3180d8ab1",
+    "result": null,
+    "status": "done"
+  }
+}
+```
+
+when the `status` is `done`, the job had finished and description of table and columns had been updated.
+
+**Next we generate sql using chat2data v2 endpoints**
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:262]:
```suggestion
**Generate SQL using the chat2data v2 endpoints**
```

**Review by nizheming — COMMENTED** [review]
LGTM

File: tidb-cloud/use-chat2query-api.md [hunk 15]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+
+#### (Optional) 4. Update a data summary for better results of `/v2/chat2data`
+
+If you want to improve the results of `/v2/chat2data` for a database, you can check and update the data summary of this database.
+
+1. To get the data summary of a database, call the `v2/dataSummaries/{data_summary_id}` endpoint in the GET method.
+
+    ```bash
+    curl --digest --user ${PUBLIC_KEY}:${PRIVATE_KEY} --request GET 'https://<region>.data.dev.tidbcloud.com/api/v1beta/app/chat2query-<ID>/endpoint/v2/dataSummaries/{data_summary_id}'\
+     --header 'content-type: application/json'
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:289]:
```suggestion
    --header 'content-type: application/json'
```

File: tidb-cloud/use-chat2query-api.md [hunk 16]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+- Replace the `<your instruction>` placeholder with the instruction you want AI to generate and execute SQL statements.
+- Replace the `<your table name, optional>` placeholder with the table name you want to query. If you do not specify a table name, AI will query all tables in the database.
 
 > **Note:**
 >
 > Each Chat2Query Data App has a rate limit of 100 requests per day. If you exceed the rate limit, the API returns a `429` error. For more quota, you can [submit a request](https://support.pingcap.com/hc/en-us/requests/new?ticket_form_id=7800003722519) to our support team.
+> API Key with the role `Chat2Query Data Context Management Role` cannot call the Chat2Data v1 endpoint.
 
-The following code example is used to find the most popular GitHub repository from `sample_data.github_events` table:
+The following code example is used to count how many users are in `sp500insight.users` table:
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:393]:
```suggestion
The following code example is used to count how many users are in the `sp500insight.users` table:
```

File: tidb-cloud/use-chat2query-api.md [hunk 17]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+When calling `/chat2data`, you need to replace the following parameters:
+
+- Replace the `${PUBLIC_KEY}` and `${PRIVATE_KEY}` placeholders with your API key.
+- Replace the `<your instruction>` placeholder with the instruction you want AI to generate and execute SQL statements.
+- Replace the `<your table name, optional>` placeholder with the table name you want to query. If you do not specify a table name, AI will query all tables in the database.
 
 > **Note:**
 >
 > Each Chat2Query Data App has a rate limit of 100 requests per day. If you exceed the rate limit, the API returns a `429` error. For more quota, you can [submit a request](https://support.pingcap.com/hc/en-us/requests/new?ticket_form_id=7800003722519) to our support team.
+> API Key with the role `Chat2Query Data Context Management Role` cannot call the Chat2Data v1 endpoint.
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:391]:
```suggestion
> API Key with the role `Chat2Query Data Summary Management Role` cannot call the Chat2Data v1 endpoint.
```

File: tidb-cloud/use-chat2query-api.md [hunk 18]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+    > If you are on the **Chat2Query** page of your cluster, you can also open the data app creation dialog by clicking **...** in the upper-right corner, choosing **Access Chat2Query via API**, and clicking **New Chat2Query Data App**.
 
-2. Click your cluster name, and then click **Chat2Query** in the left navigation pane.
-3. In the upper-right corner of Chat2Query, click **...** and select **Settings**.
-4. Enable **DataAPI** and the Chat2Query Data App is created.
-
-    > **Note:**
-    >
-    > After DataAPI is enabled for one TiDB cluster, all TiDB clusters in the same project can use the Chat2Query API.
+2. In the dialog, define a name for your Data App, choose the desired clusters as the data source, and select **Chat2Query Data App** as the Data App type. Optionally, you can also write a description for the App.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:26]:
```suggestion
2. In the dialog, define a name for your Data App, choose the desired clusters as the data sources, and select **Chat2Query Data App** as the **Data App** type. Optionally, you can also write a description for the App.
```

File: tidb-cloud/use-chat2query-api.md [hunk 19]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
-1. In the left pane of [**Data Service**](https://tidbcloud.com/console/data-service), click the name of **Chat2Query System** to view its details.
+To create an API key, perform the following steps:
+
+1. In the left pane of [**Data Service**](https://tidbcloud.com/console/data-service), click your Chat2Query Data App to view its details on the right side.
 2. In the **Authentication** area, click **Create API Key**.
-3. In the **Create API Key** dialog box, enter a description and select a role for your API key.
+3. In the **Create API Key** dialog, enter a description, and then select one of the following roles for your API key:
+
+   - `Chat2Query Admin Role`: allows the API key to manage data summaries, generate SQL statements based on provided instructions, and execute any SQL statements.
+   - `Chat2Query Data Summary Management Role`: only allows the API key to generate and update data summaries.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:43]:
```suggestion
   - `Chat2Query Data Context Management`: only allows the API key to generate and update data summaries.
```

File: tidb-cloud/use-chat2query-api.md [hunk 20]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
 
-1. In the left pane of [**Data Service**](https://tidbcloud.com/console/data-service), click the name of **Chat2Query System** to view its details.
+To create an API key, perform the following steps:
+
+1. In the left pane of [**Data Service**](https://tidbcloud.com/console/data-service), click your Chat2Query Data App to view its details on the right side.
 2. In the **Authentication** area, click **Create API Key**.
-3. In the **Create API Key** dialog box, enter a description and select a role for your API key.
+3. In the **Create API Key** dialog, enter a description, and then select one of the following roles for your API key:
+
+   - `Chat2Query Admin Role`: allows the API key to manage data summaries, generate SQL statements based on provided instructions, and execute any SQL statements.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:42]:
```suggestion
   - `Chat2Query Admin`: allows the API key to manage data summaries, generate SQL statements based on provided instructions, and execute any SQL statements.
```

File: tidb-cloud/use-chat2query-api.md [hunk 21]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
 
-    The role is used to control whether the API key can read or write data to the clusters linked to the Data App. You can select the `ReadOnly` or `ReadAndWrite` role:
+        > **Tip:**
+        >
+        > For Chat2Query API, a data summary is an analysis result of your database by AI, including your database descriptions, table descriptions, and column descriptions. By generating a data summary of your database, you can get a more accurate response when generating SQL statements by providing instructions.
 
-    - `ReadOnly`: only allows the API key to read data, such as `SELECT`, `SHOW`, `USE`, `DESC`, and `EXPLAIN` statements.
-    - `ReadAndWrite`: allows the API key to read and write data. You can use this API key to execute all SQL statements, such as DML and DDL statements.
+   - `Chat2Query SQL Generate & Execute (ReadOnly) Role`: only allows the API key to generate SQL statements based on provided instructions and execute `SELECT` SQL statements.
+   - `Chat2Query SQL Generate & Execute (ReadWrite) Role`: allows the API key to generate SQL statements based on provided instructions and execute any SQL statements.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:50]:
```suggestion
   - `Chat2Query SQL ReadOnly`: only allows the API key to generate SQL statements based on provided instructions and execute `SELECT` SQL statements.
   - `Chat2Query SQL ReadWrite`: allows the API key to generate SQL statements based on provided instructions and execute any SQL statements.
```

File: tidb-cloud/use-chat2query-api.md [hunk 22]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+    "summary_keywords": [
+        "users"
+    ],
+    "table_relationship": {}
+}
+```
+
+#### 3. Generate and execute SQL statements by calling `/v2/chat2data`
+
+When the data summary of a database is ready, you can call `/v2/chat2data` to generate and execute SQL statements by providing the data summary ID and your instructions as follows:
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:207]:
```suggestion
When the data summary of a database is ready, you can call `/v2/chat2data` to generate and execute SQL statements by providing the data summary ID and your question as follows:
```

File: tidb-cloud/use-chat2query-api.md [hunk 23]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+            }
+      }
+    }'
+    ```
+
+    In the preceding example, the request body is a JSON object with the following properties:
+
+    - `<tables_name>`: _string_. The table you want to update.
+    - `tables_name.description`: _string_. The new description you want to set for the given table.
+    - `columns`: a key-value pair to specify the target column and the column description you want to update.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:332]:
```suggestion
    - `tables_name.columns`: a key-value pair to specify the target column and the column description you want to update.
```

File: tidb-cloud/use-chat2query-api.md [hunk 24]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+                }
+            }
+      }
+    }'
+    ```
+
+    In the preceding example, the request body is a JSON object with the following properties:
+
+    - `<tables_name>`: _string_. The table you want to update.
+    - `tables_name.description`: _string_. The new description you want to set for the given table.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:331]:
```suggestion
    - `<table_name>`: _string_. The table you want to update.
    - `table_name.description`: _string_. The new description you want to set for the given table.
```

File: tidb-cloud/use-chat2query-api.md [hunk 25]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+
+You can call the `/chat2data` endpoint directly to generate and execute SQL statements. Compared with `/v2/chat2data`, `/chat2data` provides a faster response but lower performance.
+
+TiDB Cloud generates code examples to help you call an endpoint. To get the examples and run the code, see [Get the code example of an endpoint](#get-the-code-example-of-an-endpoint).
+
+When calling `/chat2data`, you need to replace the following parameters:
+
+- Replace the `${PUBLIC_KEY}` and `${PRIVATE_KEY}` placeholders with your API key.
+- Replace the `<your instruction>` placeholder with the instruction you want AI to generate and execute SQL statements.
+- Replace the `<your table name, optional>` placeholder with the table name you want to query. If you do not specify a table name, AI will query all tables in the database.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:386]: 颠倒了下顺序，和界面保持一致。
```suggestion
- Replace the `<your table name, optional>` placeholder with the table name you want to query. If you do not specify a table name, AI will query all tables in the database.
- Replace the `<your instruction>` placeholder with the instruction you want AI to generate and execute SQL statements.
```

File: tidb-cloud/use-chat2query-api.md [hunk 26]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+                }
+            }
+      }
+    }'
+    ```
+
+    In the preceding example, the request body is a JSON object with the following properties:
+
+    - `<table_name>`: _string_. The table you want to update.
+    - `table_name.description`: _string_. The new description you want to set for the given table.
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:331]:
```suggestion
    - `<table_name>.description`: _string_. The new description you want to set for the given table.
```

File: tidb-cloud/use-chat2query-api.md (hunk 19 above)
Comment (qiancai) [reply tidb-cloud/use-chat2query-api.md:43]: image

File: tidb-cloud/use-chat2query-api.md [hunk 27]
```diff
@@ -5,99 +5,402 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+TiDB Cloud provides code examples to help you quickly call Chat2Query endpoints. To get the code example of a Chat2Query endpoint, perform the following steps:
 
-- **Endpoint URL**: (read-only) the URL of the Chat2Data endpoint, which is used to call the endpoint. For example, `https://<region>.data.tidbcloud.com/api/v1beta/app/chat2query-<ID>/endpoint/chat2data`.
+1. In the left pane of the [**Data Service**](https://tidbcloud.com/console/data-service) page, click the name of a Chat2Query endpoint.
 
-- **Request Method**: (read-only) the HTTP method of the Chat2Data endpoint, which is `POST`.
+    The information for calling this endpoint is displayed on the right side, such as endpoint URL, code example, and request method.
 
-- **Timeout(ms)**: the timeout for the Chat2Data endpoint, in milliseconds.
+2. Click **Code Example**.
```
Comment (qiancai) [inline tidb-cloud/use-chat2query-api.md:81]:
```suggestion
2. Click **Show Code Example**.
```

**Review by hfxsd — APPROVED** [review]

**Review by jiajunhuang — APPROVED** [review]

File: tidb-cloud/use-chat2query-api.md [hunk 28]
```diff
@@ -5,99 +5,316 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+
+### Get the code example of an endpoint
+
+TiDB Cloud provides code examples to help you quickly call Chat2Query endpoints. To get the code example of a Chat2Query endpoint, perform the following steps:
+
+1. In the left pane of the [**Data Service**](https://tidbcloud.com/console/data-service) page, click the name of a Chat2Query endpoint.
+
+    The information for calling this endpoint is displayed on the right side, such as endpoint URL, code example, and request method.
+
+2. Click **Show Code Example**.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:81]:
```suggestion
2. Click **Show Code Example**.

```

File: tidb-cloud/use-chat2query-api.md [hunk 29]
```diff
@@ -5,99 +5,316 @@ summary: Learn how to use TiDB Cloud Chat2Query API to generate and execute SQL
...
+When calling `/v1/chat2data`, you need to replace the following parameters:
+
+- Replace the `${PUBLIC_KEY}` and `${PRIVATE_KEY}` placeholders with your API key.
+- Replace the `<your table name, optional>` placeholder with the table name you want to query. If you do not specify a table name, AI will query all tables in the database.
+- Replace the `<your instruction>` placeholder with the instruction you want AI to generate and execute SQL statements.
 
 > **Note:**
 >
 > Each Chat2Query Data App has a rate limit of 100 requests per day. If you exceed the rate limit, the API returns a `429` error. For more quota, you can [submit a request](https://support.pingcap.com/hc/en-us/requests/new?ticket_form_id=7800003722519) to our support team.
+> API Key with the role `Chat2Query Data Summary Management Role` cannot call the Chat2Data v1 endpoint.
```
Comment (hfxsd) [inline tidb-cloud/use-chat2query-api.md:305]:
```suggestion
> An API Key with the role `Chat2Query Data Summary Management Role` cannot call the Chat2Data v1 endpoint.
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**